h/templates/activity/search.html.jinja2

```diff
@@ -211,174 +211,161 @@
   {#- This form element is needed for Internet Explorer because it
       doesn't support the `form` attribute. See commit message. #}
   <form method="GET" action="{{ request.path_url }}">
     <input type="hidden" name="q" value="{{ q }}">
 
-    <h1 class="search-result-sidebar__title">{{ group.name }}</h1>
+    <h1 class="search-result-sidebar__title">{{ user.name }}</h1>
 
     <section class="search-result-sidebar__section">
-      {% if group.description %}
-        {% for paragraph in group.description.split('\n') %}
-          <p>{{ paragraph }}</p>
-        {% endfor %}
-      {% endif %}
-      <dl>
-        <dt class="search-result-sidebar__dt">
-          {% trans %}Annotations:{% endtrans %}
-        </dt>
-        <dd class="search-result-sidebar__dd">{{ stats.annotation_count }}</dd>
-        <dt class="search-result-sidebar__dt">
-          {% trans %}Created:{% endtrans %}
-        </dt>
-        <dd class="search-result-sidebar__dd">{{ group.created }}</dd>
-        {% if group_edit_url %}
-          <dt class="search-result-sidebar__dt-link">
-            <a class="link"
-               href="{{ group_edit_url }}">
-              {% trans %}Edit group{% endtrans %}
-            </a>
-          </dt>
-        {% endif %}
-        {# only show ability to leave group if group has concept of members #}
-        {% if group.members %}
-          <dt class="search-result-sidebar__dt-link">
-            <button class="link--btn js-confirm-submit"
-                    type="submit"
-                    form="search-bar"
-                    formmethod="POST"
-                    name="group_leave"
-                    value="{{ group.pubid }}"
-                    data-confirm-message="Are you sure you want to leave the group &quot;{{ group.name }}&quot;?">
-                {% trans %}Leave this group{% endtrans %}
-            </button>
-          </dt>
-        {% endif %}
-      </dl>
-    </section>
-    {{ tags_section() }}
-    {# if the group has a concept of members display members and join text #}
-    {% if group.members %} 
-      {{ group_users("Members", group.members, group.creator) }}
-  </form>
-      <h3 class="group-invite__title">
-        {% trans %}Invite new members{% endtrans %}
-      </h3>
-      <p class="search-result-sidebar__share-link">
-        {% trans %}Sharing the link lets people join this group:{% endtrans %}
-      </p>
-    {# if the group has no concept of members display moderators and share text #}
-    {% else %}
-      {{ group_users("Moderators", group.moderators, group.creator) }}
-      <h3 class="group-invite__title">
-        {% trans %}Share group{% endtrans %}
-      </h3>
-      <p class="search-result-sidebar__share-link">
-        {% trans %}Sharing the link lets people view this group:{% endtrans %}
-      </p>
+    {% if user.description %}
+      <div class="search-result-sidebar__div-group-info">
+      {% for paragraph in group.description.split('\n') %}
```

Comment:
Presumably this should be `user.description`? User profile pages are generating an error on this branch:

<img width="741" alt="screenshot 2018-02-26 12 02 43" src="https://user-images.githubusercontent.com/2458/36669528-1ae6a4f4-1aed-11e8-8091-dc77945754cb.png">

It also seems like this markup transform is something that might be useful to extract into a helper function/macro.